Frontend/src/components/ui/chart.jsx

```
@@ -40,43 +40,51 @@ const ChartContainer = React.forwardRef(({ id, className, children, config, ...p
+const ChartStyle = ({ id, config }) => {
+  const colorConfig = Object.entries(config).filter(([, config]) => config.theme || config.color);
 
   if (!colorConfig.length) {
-    return null
+    return null;
   }
 
   return (
-    (<style
+    <style
       dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)